dsew_community_profile/delphi_dsew_community_profile/pull.py

```diff
@@ -628,12 +628,14 @@ def interpolate_missing_values(dfs: DataDict) -> DataDict:
             if "val" in reindexed_group_df.columns and not reindexed_group_df["val"].isna().all():
                 reindexed_group_df["val"] = (
                     reindexed_group_df["val"]
+                    .astype(float)
                     .interpolate(method="linear", limit_area="inside")
                     .astype(float)
```

Comment:
Wondering if this cast to float at the end is unneeded now.

Reply:
Good catch. Removing the second float conversion doesn't impact test output or tests.